docs/source.rst

```diff
@@ -177,7 +177,7 @@ to make a submission to. The page will invite you to get started with your
 first submission or to log in. It should have a logo specific to the organization
 you are submitting to.
 
-|Source Interface with Javascript Disabled|
+|Source Interface with JavaScript Disabled|
```

Comment:
brand

Reply:
Definitely, we want the correct capitalization for branding here. Thanks for catching that!